app/controllers/gems_controller.rb

```diff
@@ -1,4 +1,6 @@
 class GemsController < ApplicationController
   def index
+    @gems = Gems.gems(current_user.username)
```

Comment:
As I mentioned before this is only temporary and will be replaced with RubyGems.org once that's ready. We wanted to have some sort of authentication and this is working for us right now since we're testing with users whose username is the same for Github as it is for RubyGems. 

We decided to go in this with Github as a temporary OAuth provider to make things a bit clearer for @angela4713 and @linatodi.